### PR TITLE
Bugfix/518 printers no show

### PR DESCRIPTION
**Pull request by NotExpectedYet**
Fix the loggin in PrinterCleaner and also add try and catch to client row generation.

### CODE REVIEW

File: client_src/js/printerManagementRunner.js [hunk 1]
```diff
@@ -1494,32 +1494,32 @@ function pluginListTemplate(plugin) {
                                 </div>
                                 <div class="meta">
                                     <small class="prop"><i class="fa fa-info"></i>&nbsp;<a target="_blank" href="${
-                                      plugin.page
-                                    }" title="${
+    plugin.page
```
Comment (davidzwa) [inline client_src/js/printerManagementRunner.js:1497]: Hmm consider reverting these formatting changes for the plugins

Comment (NotExpectedYet) [reply client_src/js/printerManagementRunner.js:1497]: Hu? It's just the linter running.

Comment (NotExpectedYet) [reply client_src/js/printerManagementRunner.js:1497]: I've moved them back, seems eslint and prettier don't like these templates. Keep moving them to the first line.

**Review by davidzwa — APPROVED** [review]
LGTM nice and compact

**Pull request merged**